azkaban-common/src/main/java/azkaban/AzkabanCommonModule.java

```diff
@@ -49,7 +49,7 @@
 
   public AzkabanCommonModule(Props props) {
     this.props = props;
-    this.storageImplementation = props.getString(Constants.ConfigurationKeys.AZKABAN_STORAGE_TYPE, LOCAL.name());
+    this.storageImplementation = props.getString(Constants.ConfigurationKeys.AZKABAN_STORAGE_TYPE, DATABASE.name());
```

Comment:
This line changes the default storage implementation type from `LOCAL` file system storage to `DATABASE` which is the MySQL storage used today.

`DATASTORAGE` is generic and doesn't convey anything about the implementation detail.